src/com/facebook/buck/jvm/java/intellij/DefaultIjLibraryFactory.java

```diff
@@ -165,8 +165,8 @@ public BuildRuleType getType() {
     @Override
     public void apply(
         TargetNode<AndroidPrebuiltAarDescription.Arg> targetNode, IjLibrary.Builder library) {
-      AndroidPrebuiltAarDescription.Arg arg = targetNode.getConstructorArg();
-      library.setBinaryJar(libraryFactoryResolver.getPath(arg.aar));
+      Optional<Path> jarPath = libraryFactoryResolver.getPathIfJavaLibrary(targetNode);
+      library.setBinaryJar(jarPath.get());
```

Comment:
This `.get()` will crash if a jar is not present, but that should be guaranteed by the code added in `IjProject`. Let me know if there is a better way to handle this.